apps/experiments/management/commands/duplicate_shared_pipelines.py

```diff
@@ -0,0 +1,91 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models import Count
+
+from apps.experiments.models import Experiment
+from apps.pipelines.models import Node, Pipeline
+
+
+class Command(BaseCommand):
+    help = "Find pipelines that are shared by multiple experiments and create individual copies\
+    for each experiment (except one)"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            help="Show what would be done without making changes",
+        )
+
+    def handle(self, *args, **options):
+        dry_run = options["dry_run"]
+
+        shared_pipelines = Pipeline.objects.annotate(experiment_count=Count("experiment")).filter(
```

Comment:
I'm realizing now that the default queryset will not return archived pipelines. You'll have to use `Pipeline.objects.get_all().filter(...)` to include them. I guess for the sake of data consistancy we'll want to target them as well